applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -130,15 +130,7 @@ Custom DTC overlay file for enabling a specific feature:
 * :file:`nrf91xxdk_with_nrf7002ek` - Configuration file that enables Wi-Fi scanning with nRF7002 EK.
 
 Multiple overlay files can be included to enable multiple features at the same time.
-
-.. note::
-
-   Generally, Kconfig overlays have an ``overlay-`` prefix and a :file:`.conf` extension.
-   Board-specific configuration files are placed in the :file:`boards` folder and are named as :file:`<BOARD>.conf`.
-   DTS overlay files are named the same as the build target and use the file extension :file:`.overlay`.
-   When they are placed in the :file:`boards` folder and the DTS overlay filename matches the build target,
-   the build system automatically selects and applies the overlay.
-   You can give the custom DTS overlay files as a compiler option ``-DEXTRA_DTC_OVERLAY_FILE=<dtc_filename>.overlay``.
+For more information, see :ref:`cmake_options`.
```

Comment:
The new CMake options table looks nice, but right here, the only relevant part is how to include a Kconfig overlay file. Would it be possible to have a short example of how to add an overlay Kconfig file to the build here, and then use the link for more details?

Reply:
As mentioned on teams, I want to use some form of include + replace next to these references to `cmake_options`. Let me experiment already in this PR.